components/html-block/html-block.js

```diff
@@ -155,15 +171,51 @@ class HtmlBlock extends LitElement {
 
 		this.shadowRoot.innerHTML += '<div class="d2l-html-block-rendered"></div><slot></slot>';
 
+		this.shadowRoot.querySelector('slot').addEventListener('slotchange', async e => {
+
+			const template = e.target.assignedNodes({ flatten: true })
+				.find(node => (node.nodeType === Node.ELEMENT_NODE && node.tagName === 'TEMPLATE'));
+
+			this._stamp(template);
+
+		});
+		this._renderContainer = this.shadowRoot.querySelector('.d2l-html-block-rendered');
+		this._context = { ...this._contextObserverController.values };
+	}
+
+	updated() {
+		super.updated();
+		if (this._contextObjectHasChanged()) {
```

Comment:
If none of the renderers depend on attributes, does this mean that the template won't be stamped initially?

Reply:
That's a good question. I feel like that's not the case, since this is strictly additive to what was there before. i.e. The slotchange event handler will still stamp the template. I can test this locally by stripping out the attribute on the math handler and checking if stamping still happens.

Reply:
So if no attributes were present we exploded trying to create the context controller. So I pushed a commit to fix that. I can confirm that beyond that though, the initial stamping still happens.